.github/ISSUE_TEMPLATE/product-submission-form.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: Submit a Product
+description: Submit a product to the database
+title: "[Product Submission]: <vendor> - <product>"
```

Comment:
For a quick overview, it would be helpful if the title contained also `<status>`

Reply:
Since the submitter must fill out those values manually we are encouraging only the most important items in the title.